.bumpversion_client.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 1.0.1
+current_version = v1.0.2-rc
```

Comment:
the v shouldn't be here, right?

Reply:
I'm confused here as well. I think the `v` should be added and removed at the parsing and serializing stage and not be included in the version.

But our bumpversion config needs to be properly fixed anyway (to deal with RCs etc.). So let's just merge this now and do that later.